arangod/Replication2/ReplicatedLog/Algorithms.cpp

```diff
@@ -348,7 +393,8 @@ auto algorithms::updateReplicatedLog(
       // Note that newParticipants contains the leader, while oldFollowers does
       // not.
       auto const& oldFollowers = leaderStatus->follower;
-      auto const& newParticipants = spec->currentTerm->participants;
+      auto const& newParticipants = spec->participantsConfig.participants;
+      // TODO move this calculation into updateParticipantsConfig()
```

Comment:
This TODO was introduced here, would you like to create a ticket for that?

Reply:
Done: [CINFRA-213](https://arangodb.atlassian.net/browse/CINFRA-213)